addon/components/cells/base.js

```diff
@@ -29,7 +29,7 @@ const Cell = Component.extend({
     let columnWidth = column.get('width');
 
     if (this.get('enableScaffolding') || !column) {
-      return '';
+      return null;
```

Comment:
Is `return;` ok/better as well?

Reply:
probably - though now I look at it I'm tempted to make them all `return '';` so these functions always return a string. Thoughts?